README.md

```diff
@@ -1,59 +1,29 @@
 # sparkmagic
-IPython magics for working with remote Spark clusters
 
-## Development team and Advocate
+Sparkmagic is a set of tools for interactively working with remote Spark clusters through [Livy](https://github.com/cloudera/hue/tree/master/apps/spark/java), a Spark REST server, in [Jupyter](http://jupyter.org) notebooks.
+The Sparkmagic project includes a set of IPython magics for interactively running Spark code in multiple languages, as well as some kernels that you can use to turn Jupyter into an integrated Spark environment.
 
-Subproject development team:
+## Features
 
-* Alejandro Guerrero Gonzalez, Microsoft (`@aggFTW`)
-* Auberon Lopez, Cal Poly (`@alope107`)
-* Sangeetha Shekar, Microsoft (`@sangeethashekar`)
+* Run Spark code in multiple languages against any remote Spark cluster through Livy
 
-Steering Council Advocate
+* Automatic [Zeppelin](https://zeppelin.incubator.apache.org/)-like visualization of SQL queries with the `%%sql` magic in the PySpark and Spark kernels
```

Comment:
I don't think we should mention Zeppelin. We might want to give a description of what makes them Zeppelin-like instead of just mentioning it.